authentication/auth.go

```diff
@@ -135,34 +135,61 @@ func generateClaims(postValues url.Values) (claims eqClaims) {
 	schema := postValues.Get("schema")
 	launcherSchema := surveys.FindSurveyByName(schema)
 
-	return eqClaims{
+	claims = eqClaims{
 		Claims: jwt.Claims{
 			IssuedAt: jwt.NewNumericDate(issued),
 			Expiry:   jwt.NewNumericDate(expires),
 			ID:       uuid.NewV4().String(),
 		},
 		EqID:                  launcherSchema.EqID,
 		FormType:              launcherSchema.FormType,
-		SurveyURL:			   launcherSchema.URL,
+		SurveyURL:             launcherSchema.URL,
```

Comment:
This appears as optional in the metadata as well so shouldn't this be dealt with the same way as the other optional keys?

Reply:
Made it optional too